exercises/02-storage.md

```diff
@@ -37,3 +38,49 @@ Create a server using the API and add a volume to it. Using the Cloud Control Pa
 Can you grow the size of the volume without restarting the system?
 
 Hint: Click on the question mark in the Cloud Control Panel near each volume for further instructions.
+
+---
+
+### `02.03` - Snapshot A Root Volume
+
+> Please be aware: Snapshots do not serve as a substitute for backups.
+
+Using the Cloud Control Panel, or your favorite tool, create a new server.
+
+Create a snapshot of your root volume, using our [API](https://www.cloudscale.ch/en/api/v1#volume-snapshots).
+
+> A snapshot captures the state of the disk at the moment it is taken. However, only changes that are fully written and flushed to disk before the snapshot will be included in it. In contrast, any writes that were in progress at the time of the snapshot may not be fully backed up.
```

Comment:
To avoid the use of `backup`:
```suggestion
> A snapshot captures the state of the disk at the moment it is taken. However, only changes that are fully written and flushed to disk before the snapshot will be included in it. In contrast, any writes that were in progress at the time of the snapshot may not be fully captured.
```